stage1_fly/run/main.go

```diff
@@ -129,7 +125,24 @@ func evaluateMounts(rfs string, app string, p *stage1commontypes.Pod) ([]flyMoun
 	}
 
 	// Merge command-line Mounts with ImageManifest's MountPoints
-	for _, mp := range manifestMPs {
+	var imAppManifestMPs []types.MountPoint
+	if imApp := p.Images[app].App; imApp != nil {
+		imAppManifestMPs := imApp.MountPoints
+		for _, mp := range imAppManifestMPs {
+			tuple, exists := namedVolumeMounts[mp.Name]
+			switch {
+			case exists && tuple.M.Path != mp.Path:
+				return nil, fmt.Errorf("conflicting path information from mount and mountpoint %q", mp.Name)
```

Comment:
stage1-coreos chooses to not error out in this case (and I think take the first one).

I couldn't find any part of the spec that specifically validates either choice. However, I do think the manifest is valid since both actool and the spec do not specifically say that objects with duplicate names in the spec are invalid, in which case it might make sense to run it.

I'd like another opinion on this.